### ISSUE TITLE
After expanding "Customize your keyboard shortcut" button user is unable to collapsed by pressing ENTER/SPACE key: A11y_Visual Studio Code Client_get started with accessibility feature_Keyboard

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode#241335]
[[Check out Accessibility Insights!](https://accessibilityinsights.io/)]- Identify accessibility bugs before check-in and make bug fixing faster and easier.

## GitHub Tags:
#A11yeDAD; #A11yTCS; #A11ySev4; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Jan2024; #A11yUsable; #Keyboard; #FTP; #Win32;

## Environment and OS details:
Visual Studio Code version 1.97.2
Microsoft Windows 11 Enterprise (24H2)
OS Build 26100.2605

## Repro Steps:
1.Open Visual studio code
2. TAB till Help menu item 
3. TAB till Get started with accessibility feature and press ENTER key.
5. TAB till "Customize your keyboard shortcut" button and press ENTER key.
6. Now aging press ENTER/Space to collapse it
7. Observe that after expanding it user is unable to collapsed it by pressing ENTER/Space and focus is on same control. 
## Actual:
After expanding "Customize your keyboard shortcut" button user is unable to collapsed by pressing ENTER/SPACE key when focus is on same control.
Note:  issue is with all similar buttons present under Get started with accessibility feature section. 

## Expected:
After expanding "Customize your keyboard shortcut" button user should be able to collapse by pressing ENTER/SPACE key.

## User Impact:
Keyboard only user will face difficulty because after expanding "Customize your keyboard shortcut" button user is unable to collapsed by pressing ENTER/SPACE key.

## Attachment:

**Comment by msy0991** [comment]
[문서.docx](https://github.com/user-attachments/files/18943331/default.docx)

**Comment by bhavyaus** [comment]
We use `Tab`  to navigate betweens steps and `Enter` to expand each step. Pressing `Tab` will navigate out of the selected section.
